node/pkg/governor/mainnet_tokens_test.go

```diff
@@ -88,3 +90,87 @@ func TestTokenListEmptyCoinGeckoId(t *testing.T) {
 		assert.Greater(t, len(tokenConfigEntry.coinGeckoId), 0)
 	}
 }
+
+// If not true, then there exists some chain that Wormhole governs yet has no governed tokens. That means assets
+// native to the chain are not actually governed and can be transferred across all other supported chains without
+// increasing their Governor limit. We don't want this.
+func TestAllGovernedChainsHaveGovernedAssets(t *testing.T) {
+
+	// Build a list of chains that are not governed we will filter these out. Compare with `mainnet_chains.go`
+	// When a new ChainID is added, it must either be added to this list or else have entries populated in
+	// one of the list of tokens accessed by `tokenList()`.
+	ignoredChains := map[vaa.ChainID]bool{
```

Comment:
Can node/pkg/governor/mainnet_chains.go be used instead? Seems like this would require maintaining any time a new chain is added, while this is the source of Governor truth.